### ISSUE TITLE
Expected error when running /drivebackup nextbackup

### ISSUE THREAD

**Issue by DumbCaveSpider** [MaxMaeder/DriveBackupV2#41]
I received this error when running **/drivebackup nextbackup**:
```
[03:35:47 INFO]: Player issued server command: /drivebackup nextbackup
[03:35:47 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'drivebackup' in plugin DriveBackupV2 v1.3.9
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.16.5.jar:git-Paper-586]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.16.5.jar:git-Paper-586]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:810) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.PlayerConnection.handleCommand(PlayerConnection.java:2168) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.PlayerConnection.c(PlayerConnection.java:1983) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1936) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.PacketPlayInChat.a(PacketPlayInChat.java:50) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.PacketPlayInChat.a(PacketPlayInChat.java:8) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1252) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1245) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1206) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1120) ~[patched_1.16.5.jar:git-Paper-586]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-586]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
        at ratismal.drivebackup.UploadThread.getNextAutoBackup(UploadThread.java:553) ~[?:?]
        at ratismal.drivebackup.handler.CommandHandler.onCommand(CommandHandler.java:108) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.16.5.jar:git-Paper-586]
        ... 19 more
```
I've setup a Onedrive backup and my current config is set by default when this error appeared
My server version is **Paper 1.16.5**. Any solution?

**Comment by rhld16** [comment]
Has it happened since?

**Comment by MaxMaeder** [comment]
The `/nextbackup` command is known to be unstable and buggy. I haven't gotten around to fixing it, since backups still happen when they are supposed to.

**Comment by DumbCaveSpider** [comment]
Well it somehow got `/nextbackup` fixed so it might happened again anytime. For now, it's working now.

**Comment by rhld16** [comment]
Should be fixed with 51c52b2
